query/lambda/lambda_function_in_vpc.sql

```diff
@@ -2,8 +2,11 @@ select
   -- Required Columns
   arn as resource,
   case
-    when vpc_id is null then 'alarm'
-    else 'ok'
+    when vpc_id is null then 'ok'
+    else case
+      when jsonb_array_length(vpc_subnet_ids) > 1 then 'ok'
```

Comment:
For this requirement, we took the description from its AWS config rule, [lambda-inside-vpc](https://docs.aws.amazon.com/config/latest/developerguide/lambda-inside-vpc.html). In there, I don't believe it asks for more than 1 subnet, so is there a case that this code covers?